webpack/assets/javascripts/react_app/components/Layout/LayoutSessionStorage.js

```diff
@@ -1,4 +1,4 @@
 import { getValue } from '../../common/SessionStorage';
 
 export const getIsNavbarCollapsed = () =>
-  !!getValue(`["navCollapsed","pinnedPath"]`);
+  !!getValue(`["navCollapsed","pinnedPath"]`)?.navCollapsed;
```

Comment:
Optional chaining party! :tada: 

Prefer adding parenthesis to avoid ambiguity (does the `!!` apply to everything?)
```suggestion
  !!(getValue(`["navCollapsed","pinnedPath"]`)?.navCollapsed);
```

Reply:
no need to add `!!` operator
the `navCollapsed` is already a boolean

Reply:
> no need to add !! operator
the navCollapsed is already a boolean

before the user clicks to collapse the menu, the values are initially set to `null` [by us](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/common/SessionStorage.js#L9)
I guess that's why `!!` was used,to make sure it's a boolean.

Reply:
works for me without the `!!`

Reply:
ok lets do this.. since it's only being used to [set a className](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/Layout/components/LayoutContainer.js#L9) it's indeed not important if it's `null` or `false`

Reply:
oh no.. removing it just failed 4 snapshots in [travis](https://travis-ci.org/github/theforeman/foreman/jobs/709918165#L2963)
showing `undefined` instead of `false`.. I am reverting it so the snapshot will stay the same

@jeremylenz I tried to add those brackets you ask for, but the prettier linter is annoyed by that and ask to remove it :(

Reply:
If we can't ignore the lint rule I'd go for updating the snapshots.  But either way this isn't a big enough deal to be a blocker, if it has to remain without parenthesis then I guess I'll live with it :man_shrugging:

Reply:
I prefer a snapshot with the prop as `false` instead of `undefined`
because it is not clear whether the prop wasn't passed at all, or was it pass as `undefined`? 
(in our case it was passed as `undefined`)